embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/config/source/CombinedRemoteConfigSource.kt

```diff
@@ -28,6 +28,15 @@ class CombinedRemoteConfigSource(
     }
 
     private fun attemptConfigRequest() {
-        httpSource.getConfig()?.let(store::saveResponse)
+        httpSource.getConfig()?.let {
```

Comment:
Any worry that we might purge but not succeed? I wonder if we can overwrite, and purge only if the write fails?